### ISSUE TITLE
Reproducing the problem on the coco dataset

### ISSUE THREAD

**Issue by Cocomuo** [yliu1229/AlignSeg#4]
Hello:
When I replicated the coco dataset, the accuracy was 33.2, which is not consistent with the 35.1 in the paper. But I directly used the pre-trained model you provided to reproduce the same results as in the paper. I noticed that there is a "curated" split introduced by IIC in REDEME for the coco dataset, I am not sure how to use it for this file, can you give me some guidance? Or how I should download the right coco dataset.
Looking forward to your reply!

**Comment by yliu1229** [comment]
Hi,

Please kindly refer to the orginal paper of IIC for details of the curated split:
  -- X. Ji, A. Vedaldi, and J. F. Henriques, “Invariant information clustering for unsupervised image classification and segmentation,” in Proc. IEEE/CVF Int. Conf. Comput. Vis., Seoul, South Korea, 2019, pp. 9864–9873.

If you only want a 35+ mIoU result on COCO-Stuff, you can simply train a model with 3 layers without using the mentioned curated split.

Cheers.

**Comment by Cocomuo** [comment]
Thanks for your reply!
Now that I have modified num_decode_layers:3 and trained on coco for 10 epochs, the accuracy is only 33.23, which is not the same as the accuracy in the paper. What else do I need to change to achieve the same accuracy?
Looking forward to your reply.

**Comment by yliu1229** [comment]
You can check the [eval_coco27_config.yml](https://github.com/yliu1229/AlignSeg/blob/main/configs/eval_coco27_config.yml) for details, this is the config file for evaluating COCO-Stuff with the pre-trained model I released, it reaches 35+ without using the curated split.
You should be able to reproduce the result as well :)

**Comment by yliu1229** [comment]
You can check the [eval_coco27_config.yml](https://github.com/yliu1229/AlignSeg/blob/main/configs/eval_coco27_config.yml) for details, this is the config file for evaluating COCO-Stuff with the pre-trained model I released, it reaches 35+ without using the curated split.
You should be able to reproduce the result as well :)

**Comment by Cocomuo** [comment]
Thank you for your reply: 
I carefully compared the eval_coco27_config.yml and train_coco_config.yml configuration files, and found out whether I need to change the dataset_name to "coco-all" in addition to changing the num_decode_layers to 3. Because it confuses me that for coco dataset, in train.py and sup_overcluster.py, both enter different dataset classes. How should I modify it to achieve the 35.1 precision in the paper?

![Image](https://github.com/user-attachments/assets/f886c8f9-468a-49b3-ae5f-90ad3f665005)
